src/js/models/v2/search/LocationEntity.js

```diff
@@ -40,7 +40,8 @@ const LocationEntity = {
             city_name: item.city_name || '--',
             zip_code: item.zip_code || '--',
             original_cd: item.original_cd || '--',
-            current_cd: item.current_cd || '--'
+            current_cd: item.current_cd || '--',
+            county_fips: item.county_fips || '--'
```

Comment:
you could slice(2) directly on the data that we get from the api.  i can't think of anywhere we would need the fips code to include the additional two characters in the front

Reply:
Fixed and added state fips as well.